### PR TITLE
docs: fix typos in effect 

### PR DESCRIPTION
**Pull request by sergeysova**

### CODE REVIEW

File: docs/api/effector/Effect.md [hunk 1]
```diff
@@ -69,12 +69,14 @@ const data = await getUser({id: 2}) // handle promise
 
 ### <a id='use'></a>[`use(thunk)`](#use)
 
-Setup function to call when effect trigger.
+Provides a function, which will be called when an effect is triggered.
 
-Replaces previous function inside.
+It will replase the previous function inside if there is one,
```
Comment (doasync) [inline docs/api/effector/Effect.md]:
```suggestion
It will replace the previous function inside (if any).
```

File: docs/api/effector/Effect.md [hunk 2]
```diff
@@ -69,12 +69,14 @@ const data = await getUser({id: 2}) // handle promise
...
 
 #### Arguments
 
+There can only be one argument and the function receives `params`
+
 (_`thunk`_): Function, that receive arguments passed to effect call
```
Comment (doasync) [inline docs/api/effector/Effect.md]:
```suggestion
(_`thunk`_): Function, that receives the first argument passed to an effect call
```

File: docs/api/effector/Effect.md [hunk 3]
```diff
@@ -69,12 +69,14 @@ const data = await getUser({id: 2}) // handle promise
...
 
-Setup function to call when effect trigger.
+Provides a function, which will be called when an effect is triggered.
 
-Replaces previous function inside.
+It will replase the previous function inside if there is one,
 
 #### Arguments
 
+There can only be one argument and the function receives `params`
```
Comment (doasync) [inline docs/api/effector/Effect.md]:
```suggestion
```

File: docs/api/effector/Effect.md [hunk 4]
```diff
@@ -166,8 +168,8 @@ _Event_ triggered when promise from _thunk_ is *rejected* or thunk throws.
 
 Event triggered with object of `params` and `error`:
 
-(_`params`_): Arguments passed to effect call
-(_`error`_): Error catched from thunk
+(_`params`_): An argument passed to effect call
+(_`error`_): An error catched from thunk
```
Comment (doasync) [inline docs/api/effector/Effect.md]:
```suggestion
(_`error`_): An error catched from the thunk
```

File: docs/api/effector/Effect.md [hunk 5]
```diff
@@ -102,11 +104,11 @@ Subscribe to effect calls.
 
 #### Arguments
 
-(_`watcher`_): Listener that receives `params` and `effect name`
+(_`watcher`_): Function that receives `params` and `effect name`
```
Comment (doasync) [inline docs/api/effector/Effect.md]:
```suggestion
(_`watcher`_): A function that receives `params` and `effect name`
```

File: docs/api/effector/Effect.md [hunk 6]
```diff
@@ -102,11 +104,11 @@ Subscribe to effect calls.
...
 
 #### Returns
 
-(_`Subscription`_): Unsubscribe function
+(_`Subscription`_): Function that unsubscribes the watcher
```
Comment (doasync) [inline docs/api/effector/Effect.md]:
```suggestion
(_`Subscription`_): A function that unsubscribes the watcher
```

**Comment by sergeysova** [comment]
@doasync thank you

**Pull request merged**